### PR TITLE
Fix issue #611 

### PR DESCRIPTION
**Pull request by vulnbe**
This PR fixes bug described here https://github.com/greenbone/gvmd/issues/611

- [ ] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gvmd/blob/master/CHANGELOG.md)

### CODE REVIEW

**Comment by timopollmeier** [comment]
I don't see why the `table_schema = 'public'` is evaluated before the `pg_get_serial_sequence` call in the `WITH` clause but not the original `SELECT`, so I'm not sure if it's safe to assume this is always the case.

Wouldn't it be safer to only select table_name and column_name (filtered by table_schema) in the `WITH` clause and then use `pg_get_serial_sequence` on those rows?

For example:
```
WITH table_columns AS (SELECT table_name, column_name FROM information_schema.columns WHERE table_schema = 'public')
SELECT *, pg_get_serial_sequence (table_name, column_name) FROM table_columns
WHERE pg_get_serial_sequence (table_name, column_name) IS NOT NULL;
```
or maybe
```
WITH table_columns AS (SELECT table_name, column_name FROM information_schema.columns WHERE table_schema = 'public'),
serial_sequences AS (SELECT *, pg_get_serial_sequence (table_name, column_name) AS p FROM table_columns)
SELECT * FROM serial_sequences WHERE p IS NOT NULL;
```

**Comment by mattmundell** [comment]
I presume that WITH clauses are always evaluated before the selected columns, but I did expect a solution like Timo's first example (because it's clearer that the pg_get_serial_sequence is done later).

Don't mind either way, but would like to see a summary of the explanation from #611 in a comment above the statement.

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Update result diff generation at delta reports [#650](https://github.com/greenbone/gvmd/pull/650)
 
 ### Fixed
+- An PostgreSQL statement order issue [#611](https://github.com/greenbone/gvmd/issues/611) has been addressed [#642](https://github.com/greenbone/gvmd/pull/642)
```
Comment (mattmundell) [inline CHANGELOG.md]:
```suggestion
- A PostgreSQL statement order issue [#611](https://github.com/greenbone/gvmd/issues/611) has been addressed [#642](https://github.com/greenbone/gvmd/pull/642)
```

**Comment by mattmundell** [comment]
Thank you!

**Comment by asmaack** [comment]
Will you merge this into gvmd-8.0 as well?
:-)

**Comment by mattmundell** [comment]
> Will you merge this into gvmd-8.0 as well?
> :-)

PR for this: https://github.com/greenbone/gvmd/pull/691

**Pull request merged**